### ISSUE TITLE
Einops Error. Shape mismatch

### ISSUE THREAD

**Issue by 1geek0** [lucidrains/phenaki-pytorch#27]
I trained the C-ViViT encoder with images first, got the vae loss down to ~0.05 and disc loss to ~0.007.

Then, I tried to fine-tune it with a gif dataset. But it returned an Einops shape mismatch error
<img width="715" alt="image" src="https://user-images.githubusercontent.com/7862820/211388373-ad570f27-40b8-4938-9039-d590f2bce198.png">

This only happens when I use a `batch_size` other than 1. The model keeps training properly with `batch_size` 1.

Are we supposed to use only 1 as `batch_size`? The sample code in the README has 4 as `batch_size`.

**Comment by lucidrains** [comment]
@1geek0 oh that is strange, do you have `train_on_images` set to `False`?

**Comment by 1geek0** [comment]
@lucidrains no. Here are my args

<img width="517" alt="image" src="https://user-images.githubusercontent.com/7862820/211407066-e5d20bc9-1330-447e-9456-5d915bd1cceb.png">

**Comment by lucidrains** [comment]
@1geek0 think i found a 🐛  try 0.0.67 !

**Comment by 1geek0** [comment]
Yes! Thanks. Training now 😄
